corehq/apps/userreports/pillow.py

```diff
@@ -70,19 +70,29 @@ def rebuild_tables_if_necessary(self):
                 migration_context = get_migration_context(connection, table_map.keys())
                 diffs = compare_metadata(migration_context, metadata)
 
-            tables_to_rebuild = get_tables_to_rebuild(diffs, table_map.keys())
+            tables_to_rebuild = set(get_tables_to_rebuild(diffs, table_map.keys()))
```

Comment:
Is there something in the `get_tables_to_rebuild` method that is causing it to return the same thing 5 times?